dpc/src/account/private_key.rs

```diff
@@ -39,9 +38,10 @@ pub struct PrivateKey<C: Parameters> {
     pub sk_prf: <C::PRF as PRF>::Seed,
     pub r_pk: <C::AccountCommitmentScheme as CommitmentScheme>::Randomness,
     pub r_pk_counter: u16,
-    // This dummy flag is set to true for use in the `inner_snark` setup.
-    #[derivative(Default(value = "true"))]
-    pub is_dummy: bool,
+    // TODO (howardwu): Move this into a `ProverKey` struct.
+    pk_sig: C::AccountSignaturePublicKey,
+    // TODO (howardwu): Move this into a `ProverKey` struct.
+    commitment_input: Vec<u8>,
```

Comment:
Why is it related to `ProverKey`? It is more like signature key or?

Reply:
This will be the key that an outsourced worker executing `DPC::execute()` can use (from authorization) to compute the transaction proof. (So no sk_sig, has pk_sig, as in the delegable DPC scheme)